src/randoop/NoExceptionCheck.java

```diff
@@ -58,7 +62,7 @@ public String toCodeStringPostStatement() {
 
   @Override
   public String toCodeStringPreStatement() {
-    return "";
+    return "// contract failure: this statement threw " + exceptionName + Globals.lineSep;
```

Comment:
You are right that the comment provides information about what was 
observed during generation.  I was suggesting that the comment should say 
that, because I think the current wording is potentially confusing to 
someone reading the test (and likewise for other implementations of 
toCodeStringPreStatement).